src/procedures/__tests__/TransferReservationOwnership.ts

```diff
@@ -109,6 +124,8 @@ describe('TransferReservationOwnership', () => {
     });
 
     test('should add a transaction to the queue to change the transfer reservation ownership', async () => {
```

Comment:
"should add a transaction to the queue to transfer ownership of the reservation"